src/styles/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)